lib/ReactViews/Workbench/Controls/ViewingControls.tsx

```diff
@@ -88,59 +88,42 @@ const ViewingControlMenuButton = styled(RawButton).attrs({
   }
 `;
 
-interface PropsType extends WithTranslation {
+interface PropsType {
   viewState: ViewState;
   item: BaseModel;
 }
 
-@observer
-class ViewingControls extends Component<
-  PropsType,
-  { isMapZoomingToCatalogItem: boolean }
-> {
-  constructor(props: any) {
-    // Required step: always call the parent class' constructor
-    super(props);
+const ViewingControls: React.FC<PropsType> = observer((props) => {
+  const { viewState, item } = props;
+  const { t } = useTranslation();
+  const [isMapZoomingToCatalogItem, setIsMapZoomingToCatalogItem] =
+    useState(false);
 
-    makeObservable(this);
-
-    // Set the state directly. Use props if necessary.
-    this.state = {
-      isMapZoomingToCatalogItem: false
+  useEffect(() => {
+    const hideMenu = () => {
+      runInAction(() => {
+        viewState.workbenchItemWithOpenControls = undefined;
```

Comment:
Suggestion: If not too difficult are you able to make this a local state instead? `workbenchItemWithOpenControls` doesn't get used anywhere else.